docker/sandbox/Dockerfile

```diff
@@ -70,6 +77,9 @@ COPY --from=base_ /flyteorg/ /flyteorg/
 # Copy entrypoints
 COPY docker/sandbox/flyte-entrypoint-dind.sh /flyteorg/bin/flyte-entrypoint.sh
 
+# Copy kustomization template 
+COPY docker/sandbox/kustomization.yaml.tmpl /opt/flyteorg/share/deployment/kustomization.yaml
```

Comment:
How about copying into /flyteorg/share directly and installing via kubectl apply -k instead of -f?